docker-compose.yaml

```diff
@@ -62,6 +62,7 @@ services:
     environment:
       ENV_FILE_PATH: .env.local
     volumes:
+      - "${CLIENT_VOLUME_DIR:-./config_volume}:/usr/src/app/config_volume"
```

Comment:
To extract some files to make them easily visible by developers/users since this is an example to learn (and not prod code)